src/GeneralStateTestsFiller/stBadOpcode/badOpcodeSLOADBYTESFiller.json

```diff
@@ -0,0 +1,65 @@
+{
+    "badOpcodeSLOADBYTES" : {
+        "env" : {
+            "currentCoinbase" : "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x20000",
+            "currentGasLimit" : "1000000",
+            "currentNumber" : "1",
+            "currentTimestamp" : "1000",
+            "previousHash" : "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+        "expect" : [
+            {
+                "indexes" : {
+                    "data" : -1,
+                    "gas" : 0,
+                    "value" : -1
+                },
+                "network" : ["ALL"],
+                "result" : {
+                    "095e7baea6a6c7c4c2dfeb977efac326af552d87" : {
+                        "code": "SLOADBYTES: key, mstart, msize",
+                        "code" : "0x 6001 6000 6001 e1 6001 6002 55",
```

Comment:
I thought we agreed to make correct json schemas. so instead name the fields like 
``` "//code1" :  ```
or 
``` "//comment1" : ```